MyArrayList/Program.cs

```diff
@@ -112,6 +125,7 @@ public void MinValue(bool index)
         }
         Console.WriteLine($"Maximum value is {min}");
     }
+    
     // Reverses List 
     public void ReverseList()
```

Comment:
You already used `Array.Reverse()` in `Sort()` method, guess same could be done here. But it is fine as far as we consider this as a loops usage practice.